design/helm.md

```diff
@@ -0,0 +1,145 @@
+# Helm Addons
+
+To be an addon in a Kurl cluster, they must be contributed into the repo and maintained across new breaking changes and new versions.
+This proposal will give users the flexibility to specify their own addons through a helm chart support in the Kurl installer spec.
+
+
+## Goals
+- Allow any Helm chart to be specified as part of a Kurl spec.
```

Comment:
Will we support helm2 or just helm3? Would imagine the latter?

Reply:
Yeah, I think at this point supporting Helm 2 would just be sending the wrong message to new customers, plus they stopped providing any security updates.